internal/server/admin/consumer_group_test.go

```diff
@@ -0,0 +1,38 @@
+package admin
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/gavv/httpexpect/v2"
+	"github.com/google/uuid"
+	v1 "github.com/kong/koko/internal/gen/grpc/kong/admin/model/v1"
+)
+
+func TestConsumerGroupIsNotExposed(t *testing.T) {
+	s, cleanup := setup(t)
+	defer cleanup()
+	c := httpexpect.Default(t, s.URL)
+
+	t.Run("cannot GET consumer-groups", func(t *testing.T) {
+		cg := &v1.ConsumerGroup{
+			Name: "foo",
+		}
+		res := c.GET("/v1/consumer-groups").WithJSON(cg).Expect()
```

Comment:
i know this is a placeholder, but GET typically don't have a body:
```suggestion
		res := c.GET("/v1/consumer-groups").Expect()
```

Reply:
totally!